pkgs/development/python-modules/mongomock/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, pbr
+, sentinels
+, six
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "mongomock";
+  version = "3.23.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1pdh4pj5n6dsaqy98q40wig5y6imfs1p043cgkaaw8f2hxy5x56r";
+  };
+
+  propagatedBuildInputs = [
+    setuptools
+    pbr
+    sentinels
+    six
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+
+  pythonImportsCheck = [ "mongomock" ];
+
+  meta = with lib; {
+    homepage = "https://pypi.org/project/mongomock";
+    description = "Fake pymongo stub for testing simple MongoDB-dependent code";
+    license = licenses.bsdOriginal;
```

The license looks like [3-Clause BSD](https://github.com/mongomock/mongomock/blob/develop/LICENSE).

```suggestion
    license = licenses.bsd3;
```